src/Common/Core/Test/project.json

```diff
@@ -2,7 +2,7 @@
   "dependencies": {
     "FluentAssertions": "4.6.1",
     "MicroBuild.Core": "0.2.0",
-    "Newtonsoft.Json": "8.0.3",
+    "Newtonsoft.Json": "9.0.1",
```

Comment:
No, just being paranoid, remembering the issues we had with NTVS and Newtonsoft.Json.